### ISSUE TITLE
Can't render the form with defaultfields has been set.

### ISSUE THREAD

**Issue by wudoz** [kevinchappell/formBuilder#102]
I try to set some defaultFields as below. I click the reander form button before I drop any form element into textarea ,it can't render any forms ,but after put an element then it works:
```
<script>
        jQuery(document).ready(function ($) {
            'use strict';
            var template = document.getElementById('fb-template'),
              formContainer = document.getElementById('rendered-form'),
              renderBtn = document.getElementById('render-form-button');
            var options = {
                fieldRemoveWarn: true,
                defaultFields: [{
                    label: 'Form Name',
                    name: 'fb_forname',
                    required: true,
                    description: 'Enter a Form Name !',
                    maxlength: 200,
                    className: "form-control",
                    type: 'text'
                }, {
                    description: "",
                    className: "btn btn-primary",
                    label: "Submit",
                    name: "fb_formsubmit",
                    required: "false",
                    type: "button"
                }]
            };
            $(template).formBuilder(options);
            $(renderBtn).click(function (e) {
                e.preventDefault();
                $(template).formRender({
                    container: formContainer
                });
            });
        });
    </script>
```

**Comment by ivuorinen** [comment]
[Here's codepen with your script](http://codepen.io/anon/pen/RaKggw) and all required dependencies (can be seen under "Settings" in the top panel).

Looks like it works, am I missing something?

**Comment by wudoz** [comment]
hi @ivuorinen ,
I check youe codepen script, but when I click the "render" button it didn't work.

**Comment by kevinchappell** [comment]
I wasn't sure if default fields should be saved to the new form right away. This is a simple fix I'll make now.

**Comment by kevinchappell** [comment]
@ivuorinen your codepen was just missing the form-render.js script.

@wudoz This bug is resolved in #103
